docs/telemetry.md

```diff
@@ -64,15 +64,15 @@ The event ping contains a list of events ([see event format on readthedocs.io](h
 (*) This event is sent at the end of every session.
 
 ### Browser Overlay
-| Event                                  | category | method                | object       | value                    | extras.       |
+| Event                                  | category | method                | object       | value                    | extra.       |
 |----------------------------------------|----------|-----------------------|--------------|--------------------------|---------------|
 | Browser: back clicked                  | action   | click                 | menu         | back                     |               |
 | Browser: forward clicked               | action   | click                 | menu         | forward                  |               |
 | Browser: refresh clicked               | action   | click                 | menu         | refresh                  |               |
 | Turbo mode switch clicked              | action   | change                | turbo_mode   | on/off (the new value)   |               |
 | Pin site switch clicked                | action   | change                | pin_page     | on/off (the new value)   | desktop_mode* |
 | Desktop mode switch clicked            | action   | change                | desktop_mode | on/off (the new value)   |               |
-| Tile clicked                           | action   | click                 | home_tile    | bundled/custom/youtube** | tile_id***    |
+| Tile clicked                           | action   | click                 | home_tile    | bundled/custom/youtube_tile** | tile_id***    |
```

Comment:
In the note for `youtube_tile**`, it mentions this value is removed in v3.8 but that's not true. We're trying to remove it in #2091 and, as of v3.10, it's still in there and counting using the `youtube_tile` is preferred. That will probably change pending the resolution of #2091.

Do you think you could fix that up and request re-review? Alternatively, @ mention me in #2091 and I can do it as part of that bug because it's related to my work.

Reply:
Sure, I can remove the 3.8 ending at least, but I'll leave it up to #2091 to update that value if we end up removing it.